client/src/js/Navbar.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import '../css/App.css';
+
+class Navbar extends Component {
+  state = {
+    data: null
+  };
+/*
+  componentDidMount() {
```

Comment:
You can delete this commented out part for now, I don't think we'll need an API call for the Navbar

Reply:
Deletions have been made